neqo-transport/src/connection/params.rs

```diff
@@ -293,6 +296,15 @@ impl ConnectionParameters {
         self
     }
 
+    pub fn get_grease(&self) -> bool {
```

Comment:
Rust convention is to name Boolean getters `is_foo()`.  So this would be `is_greasing` or `is_greased` or `is_greasy` or something.